doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -10,6 +10,26 @@
         "#1": "Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
         "#3": "Configure your db driver and server_version in config/packages/doctrine.yaml",
-        "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name"
+        "DATABASE_URL": "mysql://symfony:password@db:3306/symfony?charset=utf8mb4&serverVersion=5.7"
```

Comment:
I'm not in favor to put the option of the database in the DSN. cf. https://github.com/symfony/recipes/pull/258#issuecomment-346142034